core/sources/domains.move

```diff
@@ -71,6 +71,19 @@ module aptos_names::domains {
         registry: Table<NameRecordKeyV1, NameRecordV1>,
     }
 
+    /// The registry for reverse lookups (aka primary names), which maps addresses to their primary name
+    /// - Users can set their primary name to a name that they own. This also forces the name to target their own address too.
+    /// - If you point your primary name to an address that isn't your own, it is no longer your primary name.
+    /// - If you mint a name while you don't have a primary name, your minted name is auto-set to be your primary name. (including minting subdomains)
+    struct ReverseLookupRegistryV1 has key, store {
```

Comment:
I think a name like `ReverseLookupPrimaryNameRegistryV1` would a better name to indicate that this map can only be used to reverse lookup primary names, not secondary ones.